src/platform/site-wide/sass/iconography.scss

```diff
@@ -93,7 +82,18 @@
 );
 
 @include make-hub(
-  $hub-name: "careers",
+  $hub-name: "education-and-training",
```

Comment:
These name changes break the non-Drupal hub pages, since they're expected to match the hub names in vagov-content frontmatter.

Reply:
Is handleizing the name the only option? This seems like an issue in other places, too; handleizing a name that a user can change means that changing the visual name of a hub requires template changes. Is there another identifier that could be used (or added to the content model)?

One hackish way of handling this in the short term is to just have duplicate `make-hub` calls with the different hub names. It's just creating css classes, having one set for education and a second set for education-and-training isn't great, but it won't break anything.

Reply:
As an FYI if/when you revisit this, these names are also mentioned in the `Hub colors` section in the [design system](https://design.va.gov/utilities/color).